### ISSUE TITLE
LLL: remove obsolete builtin methods

### ISSUE THREAD

**Issue by axic** [ethereum/solidity#1187]
These tie in with the changes done in Solidity (removing standard contracts):

```
    "(def 'gav 0x51ba59315b3a95761d0863b05ccc7a7f54703d99)"
    "(def 'config 0x661005d2720d855f1d9976f88bb10c1a3398c77f)"
    "(def 'namereg (msg config 0))"
    "(def 'coinreg (msg config 1))"
    "(def 'gavcoin (msg config 2))"
    "(def 'sendgavcoin (to value) { [32]'send [64]:to [96]:value (call allgas gavcoin 0 32 96 0 0) })"
    "(def 'regname (name) { [32]'register [64]name (call allgas namereg 0 32 64 0 0) })"
    "(def 'regcoin (name) { [32]name (call allgas coinreg 0 32 32 0 0) })"
    "(def 'regcoin (name denom) { [32]name [64]denom (call allgas coinreg 0 32 64 0 0) })"
```

Add `keccak256` alias to `sha3`, affected builtins:

```
    "(def 'sha3 (val) { [0]:val (sha3 0 32) })"
    "(def 'sha3pair (a b) { [0]:a [32]:b (sha3 0 64) })"
    "(def 'sha3trip (a b c) { [0]:a [32]:b [64]:c (sha3 0 96) })"
```

What do you think @zigguratt?

**Comment by zigguratt** [comment]
@axic That looks good to me. Those builtins are definitely obsolete. And I think `keccak256` is actually a more accurate name than `sha3`. As well, in all of the LLL source I've looked through I've never seen either `sha3pair` or `sha3trip` used. In other words, `keccak256` is a good addition to the set.

**Comment by chriseth** [comment]
@zigguratt waiting for your pull request :-)

**Comment by zigguratt** [comment]
Oh, come _on!_ It's Thanksgiving day here in Canada! :) I'm happy to make the changes, but it'll have to wait until tomorrow morning, unfortunately. Also, I'll have some questions before I can proceed.

**Comment by zigguratt** [comment]
Pull request #1198 almost complete.

**Comment by axic** [comment]
Fixed by #1198.
